### PR TITLE
gh #84 L1 dsVideoDevice updated interface changes

### PR DESCRIPTION
**Pull request by drissya-k**
Updating L1 with the newest header files available at tag 3.0.0

### CODE REVIEW

**Review by hari22yuva — APPROVED** [review]
Approved

**Comment by drissya-k** [comment]
> add stub in skeletons/src/dsVideoDevice.c

Done

File: src/test_l1_dsVideoDevice.c [hunk 1]
```diff
@@ -1651,6 +1834,10 @@ int test_l1_dsVideoDevice_register ( void )
     UT_add_test( pSuite2, "dsForceDisableHDRSupport_L1_positive" ,test_l1_dsVideoDevice_positive_dsForceDisableHDRSupport );
     UT_add_test( pSuite2, "dsGetDFC_L1_negative" ,test_l1_dsVideoDevice_negative_dsGetDFC );
     UT_add_test( pSuite2, "dsForceDisableHDRSupport_L1_negative" ,test_l1_dsVideoDevice_negative_dsForceDisableHDRSupport );
+    UT_add_test( pSuite2, "dsRegisterFrameratePreChangeCB_L1_positive" ,test_l1_dsVideoDevice_positive_dsRegisterFrameratePreChangeCB );
+    UT_add_test( pSuite2, "dsRegisterFrameratePreChangeCB_L1_negative" ,test_l1_dsVideoDevice_negative_dsRegisterFrameratePreChangeCB );
```
Comment (akhilbhas) [inline src/test_l1_dsVideoDevice.c:1838]: The tests are added to pSuite2, which is [L1 dsVideoDevice - advanced]

Can you add it to pSuite1 ?

**Review by akhilbhas — COMMENTED** [review]
Added review comments

File: src/test_l1_dsVideoDevice.c (hunk 1 above)
Comment (drissya-k) [reply src/test_l1_dsVideoDevice.c]: Done

**Review by hari22yuva — APPROVED** [review]
Approved

**Pull request merged**